src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -366,14 +379,37 @@ export const TimelineNav = ({
                   >
                     {iconVariant(itemVariant, i)}
                   </div>
-                  {tab.props.title}
+                  <div className={clsx(styles['timeline-nav__link-title'])}>
+                    {tab.props.title}
+                  </div>
+
+                  <Icon
+                    className={clsx(styles['timeline-nav__link-arrow'])}
+                    name="arrow-forward"
+                    purpose="decorative"
```

Comment:
Is this decorative? I feel like the arrows do have a meaning – that clicking on the row will switch the view to the main content panel.

Reply:
Good catch!